pkg/reconciler/route/table_test.go

```diff
@@ -1111,7 +1111,7 @@ func TestReconcile(t *testing.T) {
 				// The Route controller attaches our label to this Configuration.
 				WithConfigLabel("serving.knative.dev/route", "ingress-mutation"),
 			),
-			rev("default", "config", 1, MarkRevisionReady, WithRevName("config-00001"), WithServiceName("windemere")),
+			rev("default", "config", 1, MarkRevisionReady, WithRevName("config-00001"), WithServiceName("windermere")),
```

Comment:
I'm assuming this is a random token and anything would do, `windermere` is a place. If it's a special thing, a url identifying it would help the next unaware reader.

Reply:
those are just strings.